core/src/gl.h

```diff
@@ -270,156 +248,121 @@ typedef char            GLchar;
 #define GL_MAX_TEXTURE_SIZE             0x0D33
 #define GL_MAX_COMBINED_TEXTURE_IMAGE_UNITS 0x8B4D
 
-#ifdef PLATFORM_ANDROID
-#define GL_APICALL  __attribute__((visibility("default")))
-#define GL_APIENTRY
-#define GL_APIENTRYP GL_APIENTRY*
-#else
-#define GL_APICALL
-#define GL_APIENTRY
-#endif
-
-extern "C" {
-
-    GL_APICALL GLenum GL_APIENTRY glGetError(void);
-    GL_APICALL const GLubyte* GL_APIENTRY glGetString(GLenum name);
-
-    GL_APICALL void GL_APIENTRY glClear(GLbitfield mask);
-    GL_APICALL void GL_APIENTRY glLineWidth(GLfloat width);
-    GL_APICALL void GL_APIENTRY glViewport(GLint x, GLint y, GLsizei width, GLsizei height);
-
-    GL_APICALL void GL_APIENTRY glEnable(GLenum);
-    GL_APICALL void GL_APIENTRY glDisable(GLenum);
-    GL_APICALL void GL_APIENTRY glDepthFunc(GLenum func);
-    GL_APICALL void GL_APIENTRY glDepthMask(GLboolean flag);
-#ifdef PLATFORM_OSX
-    GL_APICALL void GL_APIENTRY glDepthRange(GLclampd n, GLclampd f);
-    GL_APICALL void GL_APIENTRY glClearDepth(GLclampd d);
-#else
-    GL_APICALL void GL_APIENTRY glDepthRangef(GLfloat n, GLfloat f);
-    GL_APICALL void GL_APIENTRY glClearDepthf(GLfloat d);
-#endif
-    GL_APICALL void GL_APIENTRY glBlendFunc(GLenum sfactor, GLenum dfactor);
-    GL_APICALL void GL_APIENTRY glStencilFunc(GLenum func, GLint ref, GLuint mask);
-    GL_APICALL void GL_APIENTRY glStencilMask(GLuint mask);
-    GL_APICALL void GL_APIENTRY glStencilOp(GLenum fail, GLenum zfail, GLenum zpass);
-    GL_APICALL void GL_APIENTRY glClearStencil(GLint s);
-    GL_APICALL void GL_APIENTRY glColorMask(GLboolean red, GLboolean green, GLboolean blue, GLboolean alpha);
-    GL_APICALL void GL_APIENTRY glCullFace(GLenum mode);
-    GL_APICALL void GL_APIENTRY glFrontFace(GLenum mode);
-    GL_APICALL void GL_APIENTRY glClearColor(GLclampf red, GLclampf green, GLclampf blue, GLclampf alpha);
-    GL_APICALL void GL_APIENTRY glGetIntegerv(GLenum pname, GLint *params );
+namespace Tangram {
+struct GL {
```

Comment:
@hjanetzek and I were discussing yesterday about `struct` vs `namespace` for `GL` here. The only thing `struct` brings here is future proofing for if we want to add any member variable (maybe some debugging).